cachi2/core/package_managers/yarn/main.py

```diff
@@ -73,7 +73,7 @@ def _check_yarn_cache(source_dir: RootedPath) -> None:
     """Check the contents of the yarn cache using 'yarn install'.
 
     :param source_dir: the directory in which the yarn command will be called.
-    :raises SubprocessCallError: if the 'yarn install' command fails.
+    :raises CalledProcessError: if the 'yarn install' command fails.
```

Comment:
In the next commit, please change all occurrences of `:raises CalledProcessError:` in `cachi2/core/package_managers/yarn/*` to `:raises YarnBerryError:` :pray: 

All the methods will be using the run_yarn_cmd function, which is changed in the next commit to raise the more specific error